src/substrate_query_interface.js

```diff
@@ -201,14 +205,15 @@ async function getSystemEvents(api, blockHash) {
 
 // Custom
 async function getActiveEraIndex(api) {
-    let stakingActiveEra = await Promise.race([
+    let activeEra = await Promise.race([
         getStakingActiveEra(api),
-        Timeout.set(TIMEOUT_TIME_MS, 'API call staking/activeEra failed.')]);
+        Timeout.set(TIMEOUT_TIME_MS,
+            'API call custom/getActiveEraIndex failed.')]);
 
     try {
-        return stakingActiveEra.toJSON()['index'];
+        return activeEra.toJSON()['index'];
     } catch (e) {
-        throw 'Function call to getStakingActiveEra failed.';
+        throw 'API call custom/getActiveEraIndex failed.';
```

Comment:
Missing console.log(). Or maybe we should remove the other console.log()'s?

Reply:
Other `console.log()`s are intended to provide more info into where the API call has failed.

Reply:
Understood